crates/forge/src/runner.rs

```diff
@@ -531,6 +540,8 @@ impl<'a> ContractRunner<'a> {
             traces,
             labeled_addresses: labeled_addresses.clone(),
             breakpoints: Default::default(),
+            // TODO
```

Comment:
todo here

Reply:
I don't think that debugging traces were collected before for invariant testing, could we offload this for another PR ?

Reply:
Added a better desc on the TODO!

Reply:
ah cool! makes sense then.